### PR TITLE
[WC-1949] Using playwright to visual test Atlas

### PR DESCRIPTION
**Pull request by leonardomendix**
In the past, we used Cypress to test all the pages in the Atlas Design System project based on changes made in Atlas on the widget-resources repository. But since we split the repository this part of the testing was disabled.

So, the idea is to bring back this testing but instead, it is using Playwright for e2e visual testing. 

This PR adds the script to run the test locally and also for the GH Actions as a new check step for new PRs.

Please add your suggestions or concerns then we can discuss, resolve, and merge this soon. 😎

### CODE REVIEW

**Review by iobuhov — COMMENTED** [review]
Great job! 
So, new action is very clean, it looks like with playwright things getting more simple.
But, let's think how we can improve docker file reuse, this probably the only change I would propose, the reset is looking good for me 👍

File: .github/workflows/AtlasVisualComparisonTest.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+name: Atlas Visual Comparison Tests
+on:
+  push:
+    branches: [ main, master ]
+  pull_request:
+    branches: [ main, master ]
+jobs:
+  playwright:
+    name: '🧪Tests - ${{ matrix.project }} - Shard ${{ matrix.shardIndex }} of ${{ matrix.shardTotal }}'
```
Comment (iobuhov) [inline .github/workflows/AtlasVisualComparisonTest.yml:9]:
```suggestion
    name: '🧪 Tests - ${{ matrix.project }} - Shard ${{ matrix.shardIndex }} of ${{ matrix.shardTotal }}'
```

File: packages/atlas/playwright.config.ts [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+        /* Collect trace when retrying the failed test. See https://playwright.dev/docs/trace-viewer */
+        trace: "on-first-retry"
+    },
+
+    /* Configure projects for major browsers */
+    projects: [
+        {
+            name: "chromium",
+            use: { ...devices["Desktop Chrome"] }
+        },
```
Comment (iobuhov) [inline packages/atlas/playwright.config.ts:39]: As I understand for now we will run only in `chromium`?

Comment (leonardomendix) [reply packages/atlas/playwright.config.ts:39]: Exactly! The same we're doing in the web widgets repository. I thought would be better to use separate jobs for other browsers. Not sure if time-wise would be a nice run for every MR for all browsers. What do you think?

File: .github/workflows/AtlasVisualComparisonTest.yml (hunk 1 above)
Comment (leonardomendix) [reply .github/workflows/AtlasVisualComparisonTest.yml]: Thanks! I have applied your suggestion.

File: packages/atlas/playwright.config.ts (hunk 2 above)
Comment (iobuhov) [reply packages/atlas/playwright.config.ts:39]: Yeah, that's fine for me. I just making it clear 👍

**Review by samuelreichert — APPROVED** [review]

**Pull request merged**